superset-frontend/src/components/AsyncSelect.jsx

```diff
@@ -42,7 +43,7 @@ const defaultProps = {
   onAsyncError: () => {},
 };
 
-class AsyncSelect extends React.PureComponent {
+class SupersetAsyncSelect extends React.PureComponent {
```

Comment:
Probably not. Did the renaming because I was trying to `import { AsyncSelect } from src/components/Select`, as noted in the `TODO` above, but decided it was out of the scope of this PR. Can revert this change.